code/game/objects/structures/watercloset.dm

```diff
@@ -376,6 +376,13 @@
 		G.use(1)
 		return
 
+	if(istype(O, /obj/item/stack/sheet/cloth))
+		var/obj/item/stack/sheet/cloth/C = O
```

Comment:
Better variable name.

Reply:
the code above it uses a single letter name as well

Reply:
ok now give it a better variable name u boomer

Reply:
like what

Reply:
cloth

Reply:
I mean hell, of all the single letter variable names to use, you choose C which we use all the time to mean "carbon". You're showing the problem with this shortened variable name trend quite well 😄